client/src/main/java/org/apache/rocketmq/client/impl/MQAdminImpl.java

```diff
@@ -381,42 +380,12 @@ public void operationComplete(ResponseFuture responseFuture) {
                     for (MessageExt msgExt : qr.getMessageList()) {
                         if (isUniqKey) {
                             if (msgExt.getMsgId().equals(key)) {
-
-                                if (messageList.size() > 0) {
```

Comment:
why delte this logic?

Reply:
i don't understand this logic is for what.
In unexpected situation, producer send msgs to broker(s) with the same uniqKey, when query msg with the key, users may want to see all the msgs.